packages/seacas/libraries/ioss/src/Ioss_Decomposition.h

```diff
@@ -133,32 +133,44 @@ namespace Ioss {
     bool distributionFactorConstant{false}; // T if all distribution factors the same value.
   };
 
-  IOSS_EXPORT size_t
-  get_all_block_ioss_element_size(const std::vector<BlockDecompositionData> &blocks);
+  class IOSS_EXPORT ElementBlockBatchOffset
+  {
+  public:
+    ElementBlockBatchOffset(const std::vector<BlockDecompositionData> &data)
+    : m_data(data) {}
+
+    ElementBlockBatchOffset()                                = delete;
+    ElementBlockBatchOffset(const ElementBlockBatchOffset &) = delete;
```

Comment:
Why is this deleted, but move constructor is defaulted?

Reply:
Ah, that was probably a typo ... the move constructor should probably be deleted because I want to use that class internally for now until I see how things shake up wih generalization to other GroupingEntity types